openedx_lti_tool_plugin/deep_linking/api/v1/views.py

```diff
@@ -0,0 +1,49 @@
+"""Django Views."""
+from django.db.models import QuerySet
+from rest_framework.mixins import ListModelMixin
+from rest_framework.pagination import PageNumberPagination
+
+from openedx_lti_tool_plugin.deep_linking.api.v1.serializers import CourseContentItemSerializer
+from openedx_lti_tool_plugin.deep_linking.api.views import DeepLinkingViewSet
+from openedx_lti_tool_plugin.models import CourseContext
+from openedx_lti_tool_plugin.utils import get_identity_claims
+
+
+class ContentItemPagination(PageNumberPagination):
+    """Content Item Pagination."""
+
+    page_size_query_param = 'page_size'
+    max_page_size = 100
+
+
+class CourseContentItemViewSet(
```

Comment:
@kuipumu Aren't authentication_classes and permission_classes necessary?

Reply:
@anfbermudezme I forgot to add the JwtAuthentication class, but there are no permission classes necessary since this API endpoint is used by unauthenticated users.